### PR TITLE
Create About Page

### PR DESCRIPTION
**Pull request by queenly-xie77**
About Page is added under Pages and App.js.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -18,6 +19,7 @@ function App() {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/Sponsors" element={<Sponsors />} />
+          <Route path="/" element={<About />} />
```
Comment (mbruckert) [inline src/App.js]: Hey Queenly! Looks great, I just noticed one problem. Make sure to change "/" to "/About" and we should be good to go. You can just commit to this branch again with the change made, and it should update the PR.

**Review by mbruckert — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (queenly-xie77) [reply src/App.js]: Hi Mark, I made the changes, but I'm not sure if it updated into this PR, could you let me know?

**Review by nategaulke — APPROVED** [review]
Looks great and to answer your earlier question, your latest commit added the "/About"

**Review by mbruckert — APPROVED** [review]
Looks great! You can merge this when you are ready.

**Pull request merged**